src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -1050,7 +1154,7 @@ def _get_gain_output_ratio_coeffs(self):
             ],
             9: [
                 1.67e-06,
-                5.9507846,
+                5.9507846 + 1e-3,
```

Comment:
What is the reason for this opposed to just having ``5.9517846``?

Reply:
Thanks for noticing it! It's a change I did for an initialization problem and forgot to remove it.